Sources/Filters.swift

```diff
@@ -89,6 +89,50 @@ struct StringFilters {
     }
   }
 
+  /// Converts camelCase to snake_case. Takes an optional Bool argument for making the string lower case,
+  /// which defaults to true
+  /// - parameter value: the value to be processed
+  /// - parameter arguments: the arguments to the function, expecting zero or one argument
+  /// - returns: the snake case string
+  /// - throws: FilterError.invalidInputType if the value parameter isn't a string
+  static func camelToSnakeCase(_ value: Any?, arguments: [Any?]) throws -> Any? {
+    let toLower = try parseBool(from: arguments, index: 0, required: false) ?? true
+    guard let string = value as? String else { throw FilterError.invalidInputType }
+
+    let snakeCase = try snakecase(string)
+    if toLower {
+      return snakeCase.lowercased()
+    }
+    return snakeCase
+  }
+
+  /// Parses filter arguments for a boolean value, where true can by any one of: "true", "yes", "1", and
+  /// false can be any one of: "false", "no", "0". If optional is true it means that the argument on the filter is
+  /// optional and it's not an error condition if the argument is missing or not the right type
+  /// - parameter arguments: an array of argument values, may be empty
+  /// - parameter index: the index in the arguments array
+  /// - parameter required: If true, the argument is required and function throws if missing.
+  ///                       If false, returns nil on missing args.
+  /// - returns: true or false if a value was parsed, or nil if it wasn't able to
+  static func parseBool(from arguments: [Any?], index: Int, required: Bool = true) throws -> Bool? {
```

Comment:
As we're gonna keep this `parseBool` method, there's no reason for it to be only limited to string filters so no reason to scope it in the `StringFilters` type. May be worth creating a dedicated type for filter helpers to put it in there (or extend one if there's already such concept in Stencil, haven't checked)